services/self-service/src/test/java/com/epam/dlab/backendapi/resources/ProjectResourceTest.java

```diff
@@ -48,83 +56,129 @@
 import static org.mockito.Mockito.verifyNoMoreInteractions;
 import static org.mockito.Mockito.when;
 
-
 public class ProjectResourceTest extends TestBase {
-	private ProjectService projectService = mock(ProjectService.class);
-	private AccessKeyService keyService = mock(AccessKeyService.class);
-
-	@Rule
-	public final ResourceTestRule resources = getResourceTestRuleInstance(
-			new ProjectResource(projectService, keyService));
-
-	@Before
-	public void setup() throws AuthenticationException {
-		authSetup();
-	}
-
-	@Test
-	public void stopProject() {
-		final Response response = resources.getJerseyTest()
-				.target("project/stop")
-				.request()
-				.header("Authorization", "Bearer " + TOKEN)
-				.post(Entity.json(getProjectActionDTO()));
-
-		assertEquals(HttpStatus.SC_ACCEPTED, response.getStatus());
-		verify(projectService).stopWithResources(any(UserInfo.class), anyList(), anyString());
-		verifyNoMoreInteractions(projectService);
-	}
-
-	@Test
-	public void startProject() {
-		final Response response = resources.getJerseyTest()
-				.target("project/start")
-				.request()
-				.header("Authorization", "Bearer " + TOKEN)
-				.post(Entity.json(getProjectActionDTO()));
-
-		assertEquals(HttpStatus.SC_ACCEPTED, response.getStatus());
-		verify(projectService).start(any(UserInfo.class), anyList(), anyString());
-		verifyNoMoreInteractions(projectService);
-	}
-
-	@Test
-	public void generate() {
-		when(keyService.generateKeys(any(UserInfo.class))).thenReturn(new KeysDTO("somePublicKey", "somePrivateKey",
-				"user"));
-
-		final Response response = resources.getJerseyTest()
-				.target("/project/keys")
-				.request()
-				.header("Authorization", "Bearer " + TOKEN)
-				.post(Entity.json(""));
-
-		assertEquals(HttpStatus.SC_OK, response.getStatus());
-		assertEquals(MediaType.APPLICATION_JSON, response.getHeaderString(HttpHeaders.CONTENT_TYPE));
-
-		verify(keyService).generateKeys(getUserInfo());
-		verifyNoMoreInteractions(keyService);
-	}
-
-	@Test
-	public void generateKeysWithException() {
-		doThrow(new DlabException("Can not generate private/public key pair due to"))
-				.when(keyService).generateKeys(any(UserInfo.class));
-
-		final Response response = resources.getJerseyTest()
-				.target("/project/keys")
-				.request()
-				.header("Authorization", "Bearer " + TOKEN)
-				.post(Entity.json(""));
-
-		assertEquals(HttpStatus.SC_INTERNAL_SERVER_ERROR, response.getStatus());
-		assertEquals(MediaType.APPLICATION_JSON, response.getHeaderString(HttpHeaders.CONTENT_TYPE));
-
-		verify(keyService).generateKeys(getUserInfo());
-		verifyNoMoreInteractions(keyService);
-	}
-
-	private ProjectActionFormDTO getProjectActionDTO() {
-		return new ProjectActionFormDTO("DLAB", Collections.singletonList("https://localhost:8083/"));
-	}
+
+    private final static String PROJECT_NAME = "DLAB";
```

Comment:
Please follow the Java constant convention **private static final**, not **private final static**